sharding-orchestration/sharding-orchestration-core/src/main/java/org/apache/shardingsphere/orchestration/constant/OrchestrationType.java

```diff
@@ -39,11 +40,6 @@
      * @return OrchestrationType enum, return {@code null} if not found
      */
     public static OrchestrationType findByValue(final String value) {
-        for (OrchestrationType each : OrchestrationType.values()) {
-            if (each.getValue().equals(value)) {
-                return each;
-            }
-        }
-        return null;
+        return Lists.newArrayList(OrchestrationType.values()).stream().filter(each -> each.getValue().equals(value)).findFirst().get();
```

Comment:
We need to avoid using guava, Arrays.asList can replace Lists.newArrayList

Reply:
> We need to avoid using guava, Arrays.asList can replace Lists.newArrayList

OK